### DISCUSSION TITLE
Filter or sorting for the squads

### DISCUSSION THREAD

**Discussion by leamsigc** [dailydotdev/daily#1662] [Feature Requests]
![image](https://github.com/user-attachments/assets/0bb3b29d-aa17-40a1-bf3d-49028e55a70c)

The same thing, that way the new squad member can see the latest, most upvoted resources of the squad

It can go 
![image](https://github.com/user-attachments/assets/166c2920-43dd-4407-ae42-7ffcba86d325)
 or a ... at the end to toggle the filsters

**Comment by idoshamun** [comment]
Hi @leamsigc,

Thanks for the suggestion. We will consider it
